### DISCUSSION TITLE
Sharing routes/tracks

### DISCUSSION THREAD

**Discussion by johnmcd88** [osmandapp/OsmAnd#15840] [General]
Hi,

Is it possible to share routes & tracks in OsmAnd with other users, similar to sharing OsmAnd pics/vids/notes? I've used things like file explorers in the past to do so, but the newer versions of Android have really clamped down on other apps accessing OsmAnd's storage.

Thanks

John

**Comment by UdenVH** [comment]
i have the same request. 

I have created a back-up of my tracks in the OSF-file format. But this seems to be a closed OSF format and i can't extract the gpx-files in this backup. 

what are the options to export/share large numbers of gpx-tracks? i generate one to three new files daily.

**Reply by mariush444** [reply]
Change osf to zip. Then most file managers can read it.

**Comment by Boothy99** [comment]
Hi @johnmcd88,
I see this is from a while back, but I use "Solid Explorer" on my android devices to access OsmAnd's folders - including Tracks, favourites, etc. Easy to copy, paste, share the files to elsewhere.

For android 13, the Tracks & other files / folders can be located at ..
Internal Memory / Android / data / net.osmand.plus / files

Hope that helps, unless you already solved it 👍

**Reply by SamJaarsma** [reply]
I do not see that folder after I planned a route

**Reply by Boothy99** [reply]
Unfortunately, with android 14, the access to the files **Internal Memory / Android / data / net.osmand.plus / files** has been hidden 😕😠 
However, OsmAnd facilitates "Share" or "Export" for Tracks (folders, or individual tracks) as well as for Favourites. See the menu beside the tracks or folders.

**Reply by SamJaarsma** [reply]
thnx that worked! It is abit hidden though...

**Comment by DmitryAlexei** [comment]
Have you tried share option ? see https://osmand.net/docs/user/personal/tracks/manage-tracks#export

**Reply by SamJaarsma** [reply]
perfect, thanks

**Reply by maphew** [reply]
@DmitryAlexei how to share track and photos too?

**Comment by Boothy99** [comment]
As another workaround for (other) hidden files, I use the shortcut "Files" by Marc Apps & Software. It's on Google Play, and opens the hidden file explorer within android.....
https://play.google.com/store/apps/details?id=com.marc.files

Handy after android 13 😉 
Read the following guide if you wish to try it, as it requires disabling the default "Files" app by Google....
https://droidwin.com/how-to-access-android-13-data-and-obb-folder-transfer-files-in-them/

**Reply by Boothy99** [reply]
@maphew 
To share tracks, open ...
My Places,
Tracks,
To select the option for sharing multiple folders, use the "top right of screen" menu & use "Select" to choose which folders (if you have them) you wish to share, then click top right option menu again to see the Share button.
Alternatively, use the option menu beside each track to share individually.

For sharing of photos, open....
My Places,
A/V Notes,
Use the Share icon at the bottom of the screen for multi-selection (clicking "A/V Notes by date" near the top of the list, selected ALL on my screen), or just use the menu at the side of an individual file to send just one.
